### ISSUE TITLE
 Error while training the model

### ISSUE THREAD

**Issue by singla-excelsior** [harvardnlp/BSO#2]
In MT DATASET i have successfully pretrained my model.Now i move to train my model 
I execute the command :

th bso_train.lua -data_file data_prep/MT/prep-train.hdf5 -val_data_file data_prep/MT/prep-val.hdf5 -savefile mixersave -num_layers 1 -rnn_size 4 -word_vec_size 4 -save_after 21 -adagrad -curriculum 0 -epochs 21 -train_from mixerpt_epoch3_*.t7 -dropout 0.2 -max_beam_size 6 -beam_size 2 -layer_etas 0.02,0.02,0.1 -mt_delt_multiple 1

but it shows error 
loading data...	
done!	
Source vocab size: 32011, Target vocab size: 22824	
Source max sent len: 283, Target max sent len: 285	
layer_etas	0.02	0.02	0.1	
loading mixerpt_epoch3_175.20.t7...	
Number of parameters: 7137368	
/home/singla/torch/install/bin/luajit: /home/singla/Desktop/Singla/BSO/margin_beam_searcher.lua:106: attempt to call method 'floor' (a nil value)
stack traceback:
	/home/singla/Desktop/Singla/BSO/margin_beam_searcher.lua:106: in function 'nextSearchStep'
	bso_train.lua:536: in function 'train_batch'
	bso_train.lua:666: in function 'train'
	bso_train.lua:953: in function 'main'
	bso_train.lua:956: in main chunk
	[C]: in function 'dofile'
	...ngla/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50

How can i fix it ? pls help
